src/Entity/Jecoute/SurveyQuestion.php

```diff
@@ -15,27 +17,27 @@
  */
 class SurveyQuestion
 {
+    use EntityIdentityTrait;
+
     /**
      * @ORM\Column(type="integer")
      * @ORM\Id
      * @ORM\GeneratedValue
      */
-    private $id;
+    protected $id;
```

Comment:
not need if you use `EntityIdentityTrait`

Reply:
This triggers a migration error because of the "unsigned" in the `EntityIdentityTrait`. So I override it.